samples/bluetooth/mesh/src/main.c

```diff
@@ -306,6 +306,7 @@ static int gen_onoff_send(bool val)
 		.app_idx = models[3].keys[0], /* Use the bound key */
 		.addr = BT_MESH_ADDR_ALL_NODES,
 		.send_ttl = BT_MESH_TTL_DEFAULT,
+		.rnd_delay = false
```

Comment:
Here and below: same as here https://github.com/zephyrproject-rtos/zephyr/pull/67440#discussion_r1450318257